### PR TITLE
Week 2 assignments

### PR DESCRIPTION
**Pull request by joeshajan**
## Week
- week 2

## Exercises
- exercise 2.1
- exercise 2.2

## Description
- Completed both the assignments 

## Type of Change
  
- [x] assignments completed
- [x] included readme file
- [ ] feat (A new feature)
- [ ] fix (A bugfix)
- [ ] docs (Documentation-only changes)
- [ ] refactor (A code change that neither fixes a bug nor adds a feature)
- [ ] perf (A code change that improves performance)
- [ ] style (Changes that do not affect the meaning of the code - whitespace, formatting, etc)

### Where should the reviewer start? :checkered_flag:
Can start from index.html file in Exercise 2.1 and  index.html file in Exercise 2.2

### CODE REVIEW

File: Week-2/Exercise 2.1/index.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    <link rel="stylesheet" href="./styles/article.css" />
+  </head>
+
+  <body>
+    <header>
+      <h1>What is Internet</h1>
+    </header>
+
+    <article>
+      <div>
```
Comment (rahulm2) [inline Week-2/Exercise 2.1/index.html:19]: Why div? its a para right

File: Week-2/Exercise 2.1/index.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        </div>
+
+        <!-- phone no -->
+        <div class="row">
+          <div class="col-25">
+            <label for="phone">Phone no</label>
+          </div>
+          <div class="col-75">
+            <input
+              type="number"
```
Comment (rahulm2) [inline Week-2/Exercise 2.1/index.html:128]: type can be tel

File: Week-2/Exercise 2.1/index.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    <link rel="stylesheet" href="./styles/form.css" />
+    <link rel="stylesheet" href="./styles/article.css" />
+  </head>
+
+  <body>
+    <header>
+      <h1>What is Internet</h1>
+    </header>
+
+    <article>
```
Comment (rahulm2) [inline Week-2/Exercise 2.1/index.html:18]: Article should have heading summary and link

**Comment by rahulm2** [comment]
@joeshajan awesome job on assignment number 2. Wuhoo!!!

File: Week-2/Exercise 2.1/index.html (hunk 1 above)
Comment (joeshajan) [reply Week-2/Exercise 2.1/index.html]: [div tag changed to p](https://github.com/pesto-students/p7-rahulm-joeshajan/pull/3/commits/05923c59e7a958a12e4d8209f271a6689a062dfd)

File: Week-2/Exercise 2.1/index.html (hunk 2 above)
Comment (joeshajan) [reply Week-2/Exercise 2.1/index.html]: [input type changed to tel](https://github.com/pesto-students/p7-rahulm-joeshajan/pull/3/commits/f34af25f955059b4fc660eca5f899463b5c9c099)

File: Week-2/Exercise 2.1/index.html (hunk 3 above)
Comment (joeshajan) [reply Week-2/Exercise 2.1/index.html:26]: [added heading summary and link](https://github.com/pesto-students/p7-rahulm-joeshajan/pull/3/commits/762030d7f6d53dbdba4a51601d3c5f728bc637c9)

**Review by rahulm2 — APPROVED** [review]
lgtm. Verified the changes

**Pull request merged**